tox.ini

```diff
@@ -7,7 +7,7 @@ envlist =
     {py27,py34,py35}-django18,
     {py27,py34,py35}-django19,
     {py27,py34,py35,py36}-django110,
-    {py27,py34,py35,py36}-django{master}
```

Comment:
Please add a note to add django 1.11 - this will be last 1.x version with py2 support.